plugins/lando-services/scripts/sql-import.sh

```diff
@@ -109,20 +109,14 @@ if [ "$WIPE" == "true" ]; then
     psql postgresql://$USER@$HOST:$PORT/postgres -c "create database $DATABASE"
   else
     # Build the SQL prefix
-    SQLSTART="mysql -h $HOST -P $PORT -u $USER ${LANDO_EXTRA_DB_IMPORT_ARGS} $DATABASE"
-
-    # Gather and destroy tables
-    TABLES=$($SQLSTART -e 'SHOW TABLES' | awk '{ print $1}' | grep -v '^Tables' || true)
-
-    # PURGE IT ALL! Drop views and tables as needed
-    for t in $TABLES; do
-      echo "Dropping $t from $DATABASE database..."
-      $SQLSTART <<-EOF
-        SET FOREIGN_KEY_CHECKS=0;
-        DROP VIEW IF EXISTS \`$t\`;
-        DROP TABLE IF EXISTS \`$t\`;
-EOF
-    done
+    SQLSTART="mysql -h $HOST -P $PORT -u $USER ${LANDO_EXTRA_DB_IMPORT_ARGS}"
+
+    # Drop and recreate database
+    lando_yellow "Dropping database ...\n"
+    $SQLSTART -e "DROP DATABASE IF EXISTS ${DATABASE}"
```

Comment:
Something about this line is failing (see the test results for the PR or try using the [db-export example](https://github.com/lando/cli/blob/main/examples/sql-export/README.md) and running the `lando db-import test.sql` on it).